apollo-cache/src/main/java/com/apollographql/apollo/internal/cache/normalized/CacheFieldValueResolver.java

```diff
@@ -31,7 +30,7 @@ public CacheFieldValueResolver(ReadableStore readableCache, Operation.Variables
 
   @SuppressWarnings({"unchecked", "TypeParameterUnusedInFormals"})
   @Override public <T> T valueFor(Record record, ResponseField field) {
-    switch (field.type()) {
+    switch (field.getType()) {
```

Comment:
Just some clean-up after the migration to Kotlin in our `apollo-api` module. More of these should be done.